libraries/provider_runit_service.rb

```diff
@@ -311,6 +304,11 @@ def default_logger_content
         #
         # Helper Resources
         #
+        def do_action(resource, action)
+          resource.run_action(action)
+          new_resource.updated_by_last_action(true) if resource.updated_by_last_action?
```

Comment:
I thought since these were resources that have their own `#converge_by`, they don't need `#updated_by_last_action`?